modules/installation-configuration-parameters.adoc

```diff
@@ -1591,8 +1603,76 @@ Additional VMware vSphere configuration parameters are described in the followin
 |====
 |Parameter|Description|Values
 
+|`platform.vsphere`
+| Parameter covers two purposes in the context of a vSphere configuration: (1) Describes your account on the cloud platform that hosts your cluster, and you use the parameter to customize the platform; (2) The parameter is optional when providing additional configuration settings for the machine pool parameters associated with compute and control plane machines.
+a|String
+[NOTE]
+====
+You can only specify one vCenter server for your {product-title} cluster.
+====
+
+|`platform.vsphere.vCenters`
+|Lists any fully-qualified hostname or IP address of a vCenter server.
+|String
+
+|`platform.vsphere.datacenters`
+|Lists and defines the datacenters where {product-title} virtual machines (VMs) operate. The list of datacenters must match the list of datacenters specified in the `failureDomains` field.
+|String
+
+|`platform.vsphere.apiVIPs`
+|Virtual IP (VIP) addresses that you configured for control plane API access.
+a|Multiple IP addresses
+
+[NOTE]
+====
+In {product-title} 4.12 and later, the `apiVIP` configuration setting is deprecated. Instead, use a list format to enter a value in the `apiVIPs` configuration setting.
+====
+
+|`platform.vsphere.failureDomains`
+|Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+|String
+
+|`platform.vsphere.region`
+|You define a region by using a tag from the `openshift-region` tag category. The tag must be attached to the vCenter datacenter.
+|String
+
+|`platform.vsphere.zone`
```

Comment:
it should under platform.vsphere.failureDomains.zone

Reply:
Updated